lms/djangoapps/ccx/migrations/0002_convert_memberships.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Convert CCX Memberships to Course Enrollments."
+        from ccx_keys.locator import CCXLocator
+        for membership in orm['ccx.CcxMembership'].objects.all():
+            #ccx = orm['ccx.CustomCourseForEdx'].objects.get(pk=membership.ccx)
+            ccx = membership.ccx
+            course_key = CCXLocator.from_course_locator(ccx.course_id, ccx.id)
+            enrollment = orm['student.CourseEnrollment'].objects.create(
+                    user=membership.student,
+                    course_id=course_key,
+            )
+            enrollment.save()
```

Comment:
Currently, if we migrate forward, rollback, and migrate forward again, this will give a key error when trying to insert again. Please make it so that it can be run multiple times.